src/game.cpp

```diff
@@ -11939,7 +11939,7 @@ bool game::slip_down( bool check_for_traps )
     ///\EFFECT_DEX decreases chances of slipping while climbing
     ///\EFFECT_STR decreases chances of slipping while climbing
     /// Not using arm strength since lifting score comes into play later
-    int slip = 100 / ( u.dex_cur + u.str_cur );
+    int slip = 100 / std::max( 1, u.dex_cur + u.str_cur );
```

Comment:
```suggestion
    int slip = 100 / std::max( 1, 1 + u.dex_cur + u.str_cur );
```
To always decrease the chance with an increase in strength or dexterity.

Reply:
This would pretty drastically reduce the chance to slip at low stats, which I don't think is something we want to do. I also don't think it makes sense to adjust the balance within this bugfix PR.